tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -3,6 +3,7 @@
 supported_backup_backends: []
 supported_volume_backends: []
 edpm_networker_deploy: false
+ipam_configured: false
```

Comment:
Nit: maybe `configure_ipam` would be a better name? Generally i think we've been naming the variables in the convention of "do something: true/false" rather than "this thing was already done so skip it: true/false".

Reply:
done